### PR TITLE
update check clusteroperators function

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-multi-nightly-4.13-upgrade-from-ec-4.13-aws-ipi-ovn-ipsec-p2-f14

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-ec-4.13-aws-ipi-ovn-hypershift-cp-p1-f4

File: ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh [hunk 1]
```diff
@@ -187,98 +187,6 @@ function run_command_oc() {
...
-    if null_version=$(${OC} get clusteroperator -o json | jq '.items[] | select(.status.versions == null) | .metadata.name') && [[ ${null_version} != "" ]]; then
-        echo >&2 "Null Version: ${null_version}"
-        (( tmp_ret += 1 ))
-    fi
-
-    echo "Make sure every operator reports correct version"
-    if incorrect_version=$(${OC} get clusteroperator --no-headers | awk -v var="${TARGET_VERSION}" '$2 != var') && [[ ${incorrect_version} != "" ]]; then
-        echo >&2 "Incorrect CO Version: ${incorrect_version}"
-        (( tmp_ret += 1 ))
-    fi
```
Comment (shellyyang1989) [inline ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh:227]: This block checks co doesn't have empty output and reports correct version. We need to keep them.

File: ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh [hunk 2]
```diff
@@ -359,7 +267,15 @@ function health_check() {
 
     #2. Check all cluster operators get stable and ready
     echo "Step #2: check all cluster operators get stable and ready"
-    wait_clusteroperators_continous_success
+    timeout 900s bash <<EOT
+until
+  oc wait clusteroperators --all --for='condition=Available=True' --timeout=30s && \
+  oc wait clusteroperators --all --for='condition=Progressing=False' --timeout=30s && \
+  oc wait clusteroperators --all --for='condition=Degraded=False' --timeout=30s;
```
Comment (shellyyang1989) [inline ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh:274]: By default, it checks on all operators, right? If we don't want to check on a special operator, is it possible to skip it?

Comment (LiangquanLi930) [reply ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh:274]: should not be able to do

**Pull request closed without merging**